ubuntu.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
+# list of package installed through apt-get 
+# (required to run this scripts and/or as dependancies for PyNE and its depedancies)
+apt_package_list="software-properties-common \
+                  python3-pip \
+                  wget \
+                  build-essential \
+                  git \
+                  cmake \
+                  gfortran \
+                  libblas-dev \
+                  liblapack-dev \
+                  libeigen3-dev \
+                  libhdf5-dev \
+                  hdf5-tools"
+
+# list of python package required for PyNE and its depedencies (installed using pip3 python package manager)
+pip_package_list="numpy \
+                  scipy \
+                  cython \
+                  nose \
+                  tables \
+                  matplotlib \
+                  jinja2 \
+                  setuptools \
+                  future"
+
+# hdf5 std directory
+hdf5_libdir=/usr/lib/x86_64-linux-gnu/hdf5/serial
+
+#!/bin/bash
+# This script contains common code for building PyNE on various Debian-derived systems
+#
+source utils.sh
+
+# system update
+sudo apt-get -y update
+sudo apt-get install -y ${apt_package_list}
+pip3 install --user ${pip_package_list}
+
+install_dir=${HOME}/opt
+mkdir -p ${install_dir}
+
+# need to put libhdf5.so on LD_LIBRARY_PATH (Making sure that LD_LIBRARY_PATH is defined first)
+if [ -z $LD_LIBRARY_PATH ]; then
+  export LD_LIBRARY_PATH="${hdf5_libdir}"
+else
+  export LD_LIBRARY_PATH="${hdf5_libdir}:$LD_LIBRARY_PATH"
+fi
+
+
+############
+### MOAB ###
+############
+# pre-setup
+cd ${install_dir}
+check_repo moab
+mkdir -p moab
+cd moab
+
+# clone and version
+git clone --branch Version5.1.0 --single-branch https://bitbucket.org/fathomteam/moab moab-repo
+cd moab-repo
+mkdir -p build
+cd build
+
+# cmake, build and install
+cmake ../ -DENABLE_HDF5=ON -DHDF5_ROOT=${hdf5_libdir} \
+          -DBUILD_SHARED_LIBS=ON \
+          -DENABLE_PYMOAB=ON \
+          -DENABLE_BLASLAPACK=OFF \
+          -DENABLE_FORTRAN=OFF \
+          -DCMAKE_INSTALL_PREFIX=${install_dir}/moab
+make
+make install
+
+# Adding MOAB/lib to $LD_LIBRARY_PATH and $LIBRARY_PATH
+export LD_LIBRARY_PATH="${install_dir}/moab/lib:$LD_LIBRARY_PATH"
```

Comment:
don't we need to check that LD_LIBRARY_PATH isn't empty?

Reply:
not because we did the check for hdf5

Reply:
I looked, but obviously not carefully enough